### PR TITLE
[skip-ci] RPM: switch default firewall to nftables on EL10+

### PR DESCRIPTION
**Pull request by lsm5**
set NETAVARK_DEFAULT_FW to nftables on EL10+.

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

**Comment by lsm5** [comment]
ELN failures can be ignored for now. They are fixing ELN to default to dnf5 as we speak.

**Comment by lsm5** [comment]
```
[root@7629c3fd2a93 /]# rpm -q netavark
netavark-1.10.1-1.20240514125744608979.pr987.119.ge7ea9b7.el10.x86_64
[root@7629c3fd2a93 /]# /usr/libexec/podman/netavark version
{
  "version": "1.11.0-dev",
  "commit": "14044b1e19f238939aa53f10d9acf02e0da11c49",
  "build_time": "2024-05-14T00:00:00+00:00",
  "target": "x86_64-unknown-linux-gnu",
  "default_fw_driver": "nftables"
}
```

Tested on centos stream 10 container image.

@Luap99 @mheon @jnovy PTAL

File: rpm/netavark.spec [hunk 1]
```diff
@@ -22,6 +22,12 @@
 # Minimum X.Y dep for aardvark-dns
 %define major_minor %((v=%{version}; echo ${v%.*}))
 
+%if %{defined rhel} && 0%{?rhel} >= 10
```
Comment (Luap99) [inline rpm/netavark.spec:25]: just to confirm the rhel macro is also defined on centos stream?

Comment (lsm5) [reply rpm/netavark.spec:25]: yes, %{rhel} is also on centos as well as Fedora ELN. But %{centos} is only on centos stream

Comment (lsm5) [reply rpm/netavark.spec:25]: i can add a comment in spec if it helps

**Comment by lsm5** [comment]
Re-pushing with a comment.

File: rpm/netavark.spec (hunk 1 above)
Comment (lsm5) [reply rpm/netavark.spec:28]: done. PTAL

**Review by Luap99 — APPROVED** [review]
/lgtm

**Pull request merged**